packages/storybook/src/migrations/update-15-5-3/ensure-webpack-package.ts

```diff
@@ -0,0 +1,42 @@
+import {
+  getProjects,
+  joinPathFragments,
+  Tree,
+  addDependenciesToPackageJson,
+} from '@nrwl/devkit';
+import { nxVersion } from '../../utils/versions';
+import { n } from '@pmmmwh/react-refresh-webpack-plugin/types/options';
+
+// Add @nrwl/webpack as needed.
+// See: https://github.com/nrwl/nx/issues/14455
+export default async function update(tree: Tree) {
+  const projects = getProjects(tree);
+  const reactPlugin = '@nrwl/react/plugins/storybook';
+  let shouldInstall = false;
+
+  for (const [, config] of projects) {
+    let sbConfigPath = joinPathFragments(config.root, '.storybook/main.ts');
```

Comment:
Ideally, instead of `joinPathFragments(config.root, '.storybook/main.ts')` it should look for the storybook target (by searching for the `@nrwl/storybook:storybook` executor) and get the `configDir`, but I guess it's overengineering in this case. So I think it looks good.